[2025-06-29T12:02:05.031+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-08-26T00:00:00+00:00 [queued]>
[2025-06-29T12:02:05.127+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-08-26T00:00:00+00:00 [queued]>
[2025-06-29T12:02:05.136+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-06-29T12:02:05.594+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-08-26 00:00:00+00:00
[2025-06-29T12:02:05.674+0000] {standard_task_runner.py:57} INFO - Started process 5810 to run task
[2025-06-29T12:02:05.714+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2022-08-26T00:00:00+00:00', '--job-id', '1205', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpv9f_ts4e']
[2025-06-29T12:02:05.745+0000] {standard_task_runner.py:85} INFO - Job 1205: Subtask delete_data_from_table
[2025-06-29T12:02:06.538+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-08-26T00:00:00+00:00 [running]> on host f561bfe7dd3a
[2025-06-29T12:02:07.303+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-08-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-26T00:00:00+00:00'
[2025-06-29T12:02:07.313+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-08-26' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-06-29T12:02:07.401+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:02:07.545+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-06-29T12:02:07.620+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-08-26' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-06-29T12:02:07.635+0000] {sql.py:427} INFO - Rows affected: 0
[2025-06-29T12:02:07.728+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20220826T000000, start_date=20250629T120205, end_date=20250629T120207
[2025-06-29T12:02:08.034+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-29T12:02:09.367+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
